test-framework/junit5/src/main/java/io/quarkus/test/junit/launcher/ArtifactLauncherProvider.java

```diff
@@ -11,7 +11,7 @@ public interface ArtifactLauncherProvider {
      * Determines whether this provider support the artifact type
      *
      */
-    boolean supportsArtifactType(String type);
+    boolean supportsArtifactType(String type, String testProfile);
```

Comment:
This is a breaking change, although it should not really affect anyone

Reply:
Hmmmm, yeah. I can look into trying to make the change in a more backwards compatible way?

Reply:
Nah, it's extremely unlikely anyone but us is using it and even if they are, this was never documented or communicated as part of the API